### ISSUE TITLE
Decorative SVG elements should be hidden from screen reader users.

### ISSUE THREAD

**Issue by cryptasian** [microsoft/BotFramework-WebChat#4989]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Send box

### What is the public URL for the website?

https://edfinancial.com

### How to reproduce the issue?

This is feedback from the accessibility company Allyant who has been testing our website and has people who use screenreaders and other assistive technologies as a part of their testing team.

![decorative-svg](https://github.com/microsoft/BotFramework-WebChat/assets/30207740/9d82500b-8dd0-4256-a8a8-e9b244ed97de)

### What do you expect?

Each decorative <svg> tag should have an aria-hidden="true" attribute.

### What actually happened?

The decorative svg tag is not hidden.

### WCAG reference

https://www.w3.org/WAI/WCAG22/Understanding/non-text-content
